tutorial.proto

```diff
@@ -106,18 +123,39 @@ message Content {
   }
 };
 
-message Title {
-  repeated InlineContent inline_contents = 1;
+message StepTitle {
```

Comment:
I was going to go leave this just be a string field in 'Step', but I figured this has its own custom html wrapping around it.